### PR TITLE
Add support for suggested changes

### PR DESCRIPTION
**Pull request by corlewis**

### CODE REVIEW

**Review by lsf37 — APPROVED** [review]

File: Isabelle Unicode for Bitbucket.user.js [hunk 1]
```diff
@@ -560,7 +563,7 @@
 
         if (format.text_has_epilogue !== true) {
             // the 'normal' case
-            span = $(line).find(format.line_textspan);
+            span = $(line).find(format.line_textspan).addBack(format.line_textspan);
```
Comment (Xaphiosis) [inline Isabelle Unicode for Bitbucket.user.js:566]: is it worth a comment what this now does, assuming any of us have any idea?

Comment (corlewis) [reply Isabelle Unicode for Bitbucket.user.js:566]: It's including the previous element (which is `line`) when trying to find an instance of `format.line_textspan`. I added it to handle the new case where `code_line` and `line_textspan` are the same element.
There's probably a nicer way to do this but it was the simplest change that I could find with my extremely incomplete jquery knowledge.

File: Isabelle Unicode for Bitbucket.user.js [hunk 2]
```diff
@@ -15,6 +15,9 @@
 
 /*
   CHANGELOG
+  v0.4.5 2023-10-26
+  - fix format for github comment view
+  - add support for github suggested changes
```
Comment (Xaphiosis) [inline Isabelle Unicode for Bitbucket.user.js:20]: Is that all? I thought you fixed three things, or do I remember wrong?
nitpick: not sure there's a "comment view", what do you mean? Do you mean the *commit* view?

**Comment by Xaphiosis** [comment]
I'm too late :(
Nice work. Still let me know of any of the comments apply.

File: Isabelle Unicode for Bitbucket.user.js (hunk 2 above)
Comment (corlewis) [reply Isabelle Unicode for Bitbucket.user.js:20]: No, should just be those two changes in this version. You might be thinking of something I did in 0.4.4 as part of yesterday's changes.

Comment view is the name that was previously given to the format for the conversation page on Github, i.e. this page we're on right now. See https://github.com/au-ts/isabelle-symbols-bitbucket-github/pull/4/files#diff-3ba464c7dd789065b8dc53c0c0f2e0ecc5fc3db50a05890d5d8b57f784ad78afR506

Comment (Xaphiosis) [reply Isabelle Unicode for Bitbucket.user.js:20]: Oh right, I guess it's "discussion view" now, but yours is historically accurate

File: Isabelle Unicode for Bitbucket.user.js (hunk 1 above)
Comment (Xaphiosis) [reply Isabelle Unicode for Bitbucket.user.js:566]: *blink*
Well done figuring that out. I can't contribute anything better.

**Pull request merged**